includes/html/pages/device/routing/isis.inc.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use App\Models\IsisAdjacency;
+use App\Models\Port;
+
+echo '
+<div>
+  <div class="panel panel-default">
+    <div class="panel-body">
+      <table class="table table-condensed table-hover" style="border-collapse:collapse;">
+        <thead>
+          <tr>
+            <th>&nbsp;</th>
+            <th>Local device</th>
+            <th>Local interface</th>
+            <th>Adjacent</th>
+            <th>System ID</th>
+            <th>Area</th>
+            <th>System type</th>
+            <th>State</th>
+            <th>Last uptime</th>
+          </tr>
+        </thead>';
+
+// FIXME: Can refer to ports, production.ERROR: Call to undefined relationship [Port] on model [App\Models\IsisAdjacency]
```

Comment:
Still relevant?

Reply:
Not anymore, I'll push updates probably tomorrow.

Reply:
Removed old commend